airbyte-commons-worker/src/test/java/io/airbyte/workers/general/ReplicationWorkerPerformanceTest.java

```diff
@@ -65,7 +66,7 @@ public class ReplicationWorkerPerformanceTest {
   public void executeOneSync() throws InterruptedException {
     final var perSource = new LimitedAirbyteSource();
     final var perDestination = new EmptyAirbyteDestination();
-    final var messageTracker = new AirbyteMessageTracker();
+    final var messageTracker = new AirbyteMessageTracker(new EnvVariableFeatureFlags());
```

Comment:
Nit in the test should we use mock? The test might be affected by the env of where the test are run.

Reply:
The tests should use a mock, and when I replace this with the new feature-flag client I'll add mocks where necessary.  

The code (and the tests) were and are already referencing the _real_ environment variables.  So the changes in this PR shouldn't have any unexpected side-effects that didn't already exist.